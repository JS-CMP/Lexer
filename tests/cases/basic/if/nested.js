if (true) {
    console.log("Outer if");
    if (true) {
        console.log("Inner if should run");
    }
}