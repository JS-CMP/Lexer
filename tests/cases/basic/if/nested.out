#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {if (JS::Any(true)){console["log"](JS::Any("Outer if"));if (JS::Any(true)){console["log"](JS::Any("Inner if should run"));};};}
